src/pages/memberDashboard/constants.tsx

```diff
@@ -0,0 +1,270 @@
+export const TopThreeInfo = [
+  {
+    name: '1',
+    image: '/images/1.png',
+    role: 'Role',
+    batch: '2021',
+    id: 0,
+  },
+  {
+    name: '2',
+    image: 'https://bit.ly/dan-abramov',
+    role: 'Role',
+    batch: '2021',
+    id: 1,
+  },
+  {
+    name: '3',
+    image: '/images/1.png',
+    role: 'Role',
+    batch: '2021',
+    id: 2,
+  },
+]
+
+export const ExcoMembersInfo = [
```

Comment:
- [ ] Remove the mock data from the page. It can be used for debugging but not for production.